packages/react/src/helpers/index.ts

```diff
@@ -0,0 +1,20 @@
+import { OperationContext, RequestPolicy } from "urql";
```

Comment:
Not really — the only difference is which package this is in (`react` vs `api-client-core`). But I'm happy to throw this into the core as well, even if it's just used in react right now.

Reply:
Oh sorry I didn't catch that it's in this other package 🤦 Fine to leave them here, though, the convention at Gadget so far has been to avoid `thing/index.ts` files and just start with `thing.ts` until we need a whole folder to make things just a little simpler / easier to find